packages/web/docs/src/app/gateway/federation-compatible-benchmarks/index.tsx

```
@@ -90,26 +89,22 @@
         </div>
         <div className="flex items-center gap-1">
-          <XIcon className="size-4" style={{ color: functionalTones.criticalDark }} /> Failed tests
+          <XIcon className="size-4" className="text-critical-dark" /> Failed tests
         </div>
       </div>
       <div className="flex items-center gap-2">
-        <div
-          className="size-2 rounded-full"
-          style={{ background: functionalTones.positiveBright }}
-        />
+        <div className="size-2 rounded-full" className="bg-positive-bright" />
```

Biome (1.9.4)

[error] This JSX property is assigned multiple times.

This attribute is assigned again here.


(lint/suspicious/noDuplicateJsxProps)